### PR TITLE
BK-4157: User Skipper to build ec2 api

### PR DESCRIPTION
**Pull request by maroun-stratoscale**
@Stratoscale/mgmt

### CODE REVIEW

File: Makefile.ec2-api [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+clean:
+	find -name *.py[co] -delete
+
+approve:
+	@echo approve
+
+assert_no_local_changes:
+	@if ! git diff --exit-code --quiet; then echo "Please commit any local changes prior to pushing"; false; fi
+	@if ! git diff --exit-code --cached --quiet; then echo "Please commit any local staged changes prior to pushing"; false; fi
+
```
Comment (shay-stratoscale) [inline Makefile.ec2-api]: consider adding a 'deploy' target, you can look at k8s for an example

File: artifacts.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+containers:
+ - stratoscale/ec2-api
```
Comment (ravid-stratoscale) [inline artifacts.yaml]: no /stratoscale

**Comment by ravid-stratoscale** [comment]
Please run check-it on the branch, there might be needed rootfs-star changes

File: artifacts.yaml (hunk 2 above)
Comment (maroun-stratoscale) [inline artifacts.yaml]: Done

**Pull request merged**